apps/microsoft-teams/frontend/src/components/config/ChannelSelection/ChannelSelection.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import ChannelSelection from './ChannelSelection';
+import { describe, expect, it, vi } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { channelSelection } from '@constants/configCopy';
+import { defaultNotification } from '@constants/defaultParams';
+
+describe('ChannelSelection component', () => {
+  it('mounts and renders the correct title and button copy when no channel is selected', () => {
+    const { unmount } = render(
+      <ChannelSelection notification={defaultNotification} handleNotificationEdit={vi.fn()} />
+    );
+
+    expect(screen.getByText(channelSelection.title)).toBeTruthy();
+    expect(screen.getByText(channelSelection.addButton)).toBeTruthy();
+    unmount();
+  });
+  it('mounts and renders an input when a channel is selected', () => {
+    render(
```

Comment:
Just purely a curiousity / for my own learning question -- why do we need to unmount in the first test but not this one?

Reply:
We learned with using vitest in AICG that if we have multiple tests in the same describe block, we need to unmount the component. So it was really just to make the second test in the suite run properly. I can address adding unmount in the second test in a follow up PR.